README.md

```diff
@@ -25,6 +25,35 @@ Mention `@Peel` in [Discord](https://discord.gg/6jXrJSyDFf) and someone from our
 
 Read the [development guidelines](doc/development.md) for instructions on running the app in development.
 
+## Testing
```

Comment:
**suggestion** prolly can just add this to the dev doc above, wdyt?

Reply:
yep for sure - will do